datalab/mlalpha/_job.py

```diff
@@ -68,6 +51,61 @@ def describe(self):
     job_yaml = yaml.safe_dump(self._info, default_flow_style=False)
     print job_yaml
 
+  @staticmethod 
+  def submit_training(job_request, job_id=None):
+    """Submit a training job.
```

Comment:
expand comment that the args dict will get expanded to 
--key=value or --key=list_item_0 --key=list_item_1, ...

Also say that the python_module "trainer/task.py" should be able to parse these args.

Wait, regular flags are not supported? So task.py cannot use just "--foo". Only key-value parameters are supported? I don't like this restriction. I also bet we don't document this restriction anywhere. 

So command lines that are not supported in datalab:
1) --foo
2) --foo 1 2 3
3) --foo=--bar=2  (because we append the key and value in two calls to .append)
4) -n
5) -n 200

I would rather allow args to be a dict or list. So then we support every kind of command line. Or we should at least document task.py arg's restrictions